src/app/home-page/recent-item-list/recent-item-list.component.ts

```diff
@@ -0,0 +1,63 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { PaginatedSearchOptions } from '../../shared/search/models/paginated-search-options.model';
+import { fadeIn, fadeInOut } from '../../shared/animations/fade';
+import { RemoteData } from '../../core/data/remote-data';
+import { PaginatedList } from '../../core/data/paginated-list.model';
+import { Item } from '../../core/shared/item.model';
+import { PaginationComponentOptions } from '../../shared/pagination/pagination-component-options.model';
+import { PaginationService } from '../../core/pagination/pagination.service';
+import { SearchService } from '../../core/shared/search/search.service';
+import { SortDirection, SortOptions } from '../../core/cache/models/sort-options.model';
+import { DefaultAppConfig } from '../../../config/default-app-config';
+import { AppConfig } from '../../../config/app-config.interface';
+import { ViewMode } from '../../core/shared/view-mode.model';
+import {
+  toDSpaceObjectListRD
+} from '../../core/shared/operators';
+import {
+  Observable,
+ } from 'rxjs';
+@Component({
+  selector: 'ds-recent-item-list',
+  templateUrl: './recent-item-list.component.html',
+  styleUrls: ['./recent-item-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  animations: [
+    fadeIn,
+    fadeInOut
+  ]
+})
+export class RecentItemListComponent implements OnInit {
+  itemRD$: Observable<RemoteData<PaginatedList<Item>>>;
+  paginationConfig: PaginationComponentOptions;
+  sortConfig: SortOptions;
+    /**
+   * The view-mode we're currently on
+   * @type {ViewMode}
+   */
+     viewMode = ViewMode.ListElement;
+  constructor(private searchService: SearchService,
+    private paginationService: PaginationService,
+   ) {
+    const appConfig: AppConfig = new DefaultAppConfig();
+    this.paginationConfig = Object.assign(new PaginationComponentOptions(), {
+      id: 'hp',
+      pageSize:appConfig.rpp,
+      currentPage: 1,
+      maxSize:1
+    });
+    this.sortConfig = new SortOptions('dc.date.accessioned', SortDirection.DESC);
```

Comment:
Might be interesting to have the sort field also configurable, in case someone would for example want to use `dc.date.available` (recently un-embargoed items would then also show up in recent submissions) or in case this field ever changes in REST
Eg under environment.homepage.recentSubmissionsSortField

Though then the same should be done for recent submission on collection page (in `src/app/collection-page/collection-page.component.ts`)

Reply:
added recentSubmissionsSortField in homepageConfig..